### PR TITLE
Use executorId instead of generating UUID for metrics

### PR DESCRIPTION
**Pull request by AlfredBroda**
To allow grouping metrics by application name instead of genarating a UUID for every executor instance use it's executorId.

### CODE REVIEW

File: metrics/graphite.go [hunk 1]
```diff
@@ -17,18 +16,18 @@ import (
 
 const graphiteConfigEnvPrefix = "allegro_executor_graphite"
 
-var metricsUUID = uuid.New()
-
-func init() {
+// Init processes the environment in search of Graphite configuration and sets up a connection
+func Init(uuid string) {
```
Comment (medzin) [inline metrics/graphite.go]: `id` for parameter name is enough.

File: metrics/graphite.go [hunk 2]
```diff
@@ -38,9 +37,10 @@ func init() {
 
 // GraphiteConfig holds basic Graphite configuration.
 type GraphiteConfig struct {
-	Host   string
-	Port   int    `default:"2003"`
-	Prefix string `default:"allegro.executor"`
+	Host        string
+	MetricsUUID string
```
Comment (medzin) [inline metrics/graphite.go]: Why did you add this parameter here?

Comment (AlfredBroda) [reply metrics/graphite.go]: You are right, I can go back to using a local var.

File: metrics/graphite.go (hunk 1 above)
Comment (AlfredBroda) [reply metrics/graphite.go]: Ok.
